Core/Core/Serialisation/BaseObjectDeserializerV2.cs

```diff
@@ -131,7 +131,7 @@ private List<(string, int)> GetClosures(string rootObjectJson)
       }
       return closureList;
     }
-    catch (Exception)
+    catch (Exception ex) when (!ex.IsFatal())
```

Comment:
Same, I know there't not part of this work..  GetClosures() exploded and we go, ahh clearly there were none :D

Reply:
Agreed, but needs some future clean-up/refactor for us to properly fix.